TombEngine/Objects/TR4/Entity/tr4_baddy.cpp

```diff
@@ -1335,7 +1335,7 @@ namespace TEN::Entities::TR4
 			}
 			else if (object.hitEffect == HitEffect::Blood)
 			{
-				DoBloodSplat(pos->x, pos->y, pos->z, 10, source.Pose.Orientation.y, pos->RoomNumber);
+				DoBloodSplat(pos->x, pos->y, pos->z, 5, source.Pose.Orientation.y, pos->RoomNumber);
```

Comment:
Why increase?

Reply:
added random and fixed roomNumber being 0 which caused water blood in my level.